client/extensions/woocommerce/app/order/order-shipping-row.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import formatCurrency from 'lib/format-currency';
+
+class OrderShippingRow extends Component {
+	static propTypes = {
+		order: PropTypes.shape( {
+			currency: PropTypes.string.isRequired,
+			shipping_tax: PropTypes.string.isRequired,
+			shipping_total: PropTypes.string.isRequired,
+		} ),
+		showTax: PropTypes.bool,
+	}
+
+	render() {
+		const { order, showTax, translate } = this.props;
+		if ( ! order ) {
+			return null;
+		}
+
+		const tax = (
+			<div className="order__details-totals-tax">
+				{ formatCurrency( order.shipping_tax, order.currency ) }
+			</div>
+		);
+
+		return (
+			<div className="order__details-total-shipping">
+				<div className="order__details-totals-label">{ translate( 'Shipping' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Shipping address' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).